scripts/docker/jenkinsAgent/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Use the Jenkins agent base image
+FROM jenkins/agent:alpine-jdk17
+
+# Set environment variables to avoid user interaction during installation
+ENV DEBIAN_FRONTEND=noninteractive
+ARG terraform_ver=1.6.0
+ARG helm_ver=3.15.4
+ARG kubectl_ver=v1.28.0
+ARG kafka_ver=3.8.0
+
+# Install required packages
+USER root
+RUN apk update && \
+    apk add --no-cache \
+    aws-cli \
+    postgresql-client \
+    unzip
+# Install Terraform
+RUN curl -fsSL "https://releases.hashicorp.com/terraform/${terraform_ver}/terraform_${terraform_ver}_linux_amd64.zip" -o terraform.zip && \
+    unzip terraform.zip && \
+    mv terraform /usr/local/bin/ && \
```

Comment:
Just in case:
Could you please make sure we don't need to run 
**chmod +x ${TOOL}** here and below :)

Reply:
I`ve added chmod +x